packages/strapi-design-system/src/v2/Breadcrumbs/CrumbSimpleMenu.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { SimpleMenu } from '../SimpleMenu';
+
+import CarretDown from '@strapi/icons/CarretDown';
+
+export const CrumbSimpleMenu = ({ children, ...props }) => (
+  <SimpleMenu noBorder icon={<CarretDown />} size="S" {...props}>
```

Comment:
SimpleMenu should be `as={IconButton}` and then height of the IconButton reduced

Reply:
```
const IconButtonCustom = styled(IconButton)`
  height: ${({ theme }) => theme.spaces[3]};
  background-color: transparent;
`;

...


<SimpleMenu as={IconButtonCustom} noBorder icon={<CarretDown />} size="S" {...props}>
```

Reply:
I'm already usign the `size="S"` variant, which visually works well for me, don't you think?

<img width="405" alt="Screenshot 2022-07-18 at 08 58 34" src="https://user-images.githubusercontent.com/2244375/179459150-64a4944b-7680-4f54-afcd-4ff0669956ec.png">